processor/src/job.d.ts

```diff
@@ -1,16 +1,22 @@
 import { Job as BullMQJob } from 'bullmq'
 import { KitspaceYaml } from './kitspaceYaml'
 
-export interface JobData {
-  giteaId: string | null
-  kitspaceYaml: KitspaceYaml
-  originalUrl: string
+export interface RepoJobData {
   defaultBranch: string
-  ownerName: string
-  repoName: string
+  giteaId: string | null
   hash: string
   inputDir: string
+  kitspaceYamlArray: Array<KitspaceYaml>
+  originalUrl: string
   outputDir: string
+  ownerName: string
+  repoName: string
+  repoDescription: string
+}
+
+export interface ProjectJobData
+  extends Omit<RepoJobData, 'kitspaceYamlArray' | 'repoDescription'> {
+  kitspaceYaml: KitspaceYaml
```

Comment:
Did prettier do this formatting? Is it being checked?

Reply:
Yes, here is the output for `yarn fmt`

```
$ yarn prettier --write
$ prettier '*.json' src/ test/ --write
.eslintrc.json 17ms
package.json 4ms
tsconfig.json 2ms
src/app.ts 23ms
src/env.ts 14ms
src/events.ts 5ms
src/giteaDB.ts 31ms
src/job.d.ts 5ms   <---------------------------- perttier formats .d.ts files too.
src/kitspaceYaml.ts 21ms
src/log.ts 4ms
src/main.ts 5ms
src/meili.ts 4ms
src/queues.ts 27ms
src/redisConnection.ts 3ms
src/s3.ts 24ms
src/shell.ts 6ms
src/tasks/addToSearch.ts 9ms
src/tasks/cleanUp.ts 4ms
src/tasks/processBOM/get_partinfo.ts 4ms
src/tasks/processBOM/index.ts 14ms
src/tasks/processGerbers/board_builder.ts 10ms
src/tasks/processGerbers/findGerberFiles.ts 9ms
src/tasks/processGerbers/index.ts 38ms
src/tasks/processIBOM/index.ts 11ms
src/tasks/processInfo.ts 3ms
src/tasks/processKicadPCB/index.ts 13ms
src/tasks/processPCB.ts 4ms
src/tasks/processReadme/index.ts 9ms
src/tasks/processReadme/renderMarkdown.ts 7ms
src/tasks/processReadme/urlTransformer.ts 10ms
src/tasks/writeKitspaceYaml.ts 8ms
src/utils.ts 15ms
src/watcher.ts 10ms
src/workers.ts 15ms
test/integration/projects.test.ts 51ms
test/unit/readme_processing.test.ts 10ms
Done in 1.14s.
```